learn-arai/app/checking/page.tsx

```diff
@@ -0,0 +1,128 @@
+'use client';
+
+import Image from 'next/image';
+
+import { FormEvent } from 'react';
+import React from 'react';
+
+import { clsx } from 'clsx';
+import type { ClassValue } from 'clsx';
+import { OTPInput, SlotProps } from 'input-otp';
+import { twMerge } from 'tailwind-merge';
+
+import sideLoginPicture from '@/public/login/teaching.jpeg';
+
+import '@/app/register/register.css';
+
+function Slot(props: SlotProps) {
+    return (
+        <div
+            className={cn(
+                'relative w-10 h-14 text-[2rem]',
+                'flex items-center justify-center',
+                'transition-all',
+                'border-border border-y border-r first:border-l first:rounded-l-md last:rounded-r-md',
+                'group-hover:border-accent-foreground/20 group-focus-within:border-accent-foreground/20',
+                'outline outline-0 outline-accent-foreground/20',
+                { 'outline-4 outline-accent-foreground': props.isActive }
+            )}
+        >
+            {props.char !== null && <div>{props.char}</div>}
+            {props.hasFakeCaret && <FakeCaret />}
+        </div>
+    );
+}
+
+// You can emulate a fake textbox caret!
+function FakeCaret() {
+    return (
+        <div className="absolute pointer-events-none inset-0 flex items-center justify-center animate-caret-blink">
+            <div className="w-px h-8 bg-white" />
+        </div>
+    );
+}
+
+// Inspired by Stripe's MFA input.
+function FakeDash() {
+    return (
+        <div className="flex w-10 justify-center items-center">
+            <div className="w-3 h-1 rounded-full bg-border" />
+        </div>
+    );
+}
+
+function cn(...inputs: ClassValue[]) {
+    return twMerge(clsx(inputs));
+}
+
+async function submitVerification(event: FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    const formData = new FormData(event.currentTarget);
+
+    const result = await fetch(
+        'http://localhost:3000/auth/email-verification',
```

Comment:
Use env NEXT_PUBLIC_BACKEND_URL instead of hardcode link (this won't work on production)

Reply:
@MonitorIizard  teach me like this,I don't know what to do

Reply:
Like this

![Using NEXT_PUBLIC_BACKEND_URL](https://github.com/learn-arai/learn-arai/assets/40863067/8ae7e924-51ca-4abf-8ba7-0a6fc7837220)

Reply:
already fixed
![Uploading image.png…]()